.ruff.toml

```diff
@@ -85,6 +86,11 @@ ignore = [
     "FBT002",  # boolean-default-value-in-function-definition
     "FBT003",  # boolean-positional-value-in-function-call
 
+    # flake8-fixme (FIX)
+    "FIX001",  # Line contains FIXME
+    "FIX003",  # Line contains XXX
+    "FIX004",  # Line contains HACK
```

Comment:
`FIX001` and `FIX003` would be good to enforce because replacing all occurrences of `FIXME` and `XXX` in comments with `TODO` makes grepping for such occurrences much simpler. I'm not so sure `FIX004` should be addressed because I don't think `HACK` is interchangeable with `TODO`.

Reply:
Just clarifying, the thought is that `HACK` is distinct from `TODO` and `NOTE` and should be allowed to be merged if present in a PR.

Reply:
I do think that `HACK` is very distinct from `TODO`, but you also mention `NOTE` which I hadn't thought of and which would be a good replacement. I therefore suggest adding comments that recommend replacing `FIXME` and `XXX` with `TODO` and `HACK` with `NOTE`.

I should point out that it is not obvious that `XXX` can be replaced with `TODO`, but `astropy` violates `FIX003` in only a single location, and it is simple to verify that in that location `TODO` is indeed appropriate.

Reply:
Done!